mynn/layers/batchnorm.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+import mygrad as mg
+
+from mynn.initializers.constant import constant
+
+class batchnorm:
+    ''' A batch normalization layer.
+
+    This class will perform an n-dimensional batch normalization operation on an (N, D, ...)
+    shaped tensor scaled by γ of shape (D, ...) and shifted by β of shape (D, ...).
+    '''
+    def __init__(self, input_shape, momentum=0.1, gamma_initializer=constant, 
```

Comment:
For now, I don't think we need to give the initializers as a parameter. See the main body of my review.